generators/guide/{{name}}/guide.md.hbs

```diff
@@ -0,0 +1,15 @@
+# {{name}}
+
+## Section name
```

Comment:
Does a `section` delineate a particular set of codemods? Like, if the guide for a version covers two unrelated updates in the latest release version, are you locked in to making both upgrades, or can you pick and choose upgrades within a given version?

If there are interdependencies between two upgrades, is there some way for us to account for that? (Can't think of an example at the moment but just thinking out loud)